src/dp_flow.c

```diff
@@ -207,6 +213,7 @@ int dp_build_flow_key(struct flow_key *key /* out */, struct rte_mbuf *m /* in *
 
 void dp_invert_flow_key(const struct flow_key *key /* in */, uint16_t l3_type /* in */, struct flow_key *inv_key /* out */)
 {
+	// since we are inverting the key, couldn't this ask key->l3_type instead? (will make more stuff easier)
 	if (l3_type == RTE_ETHER_TYPE_IPV4) {
 		inv_key->l3_src.ip4 = key->l3_dst.ip4;
 		inv_key->l3_dst.ip4 = key->l3_src.ip4;
```

Comment:
I think this is just a leftover from the times when `flow_key` had no `l3_type` and I can easily just drop the old argument

Reply:
Yes, we can drop the old argument.